test/xpu/run_test_with_skip.py

```diff
@@ -5,10 +5,16 @@
 
 
 res = 0
+fail_test = []
 
 for key in skip_dict:
     skip_list = skip_dict[key]
-    res += launch_test(key, skip_list)
+    fail = launch_test(key, skip_list)
+    res += fail
+    if fail:
+        fail_test.append(key)
+if fail_test:
```

Comment:
What is the fail_test list for?

Reply:
A summary of all UTs in CI. Easy to find failed UT.